src/components/layout/nav/MobileNav/MobileNav.styled.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box } from '@mui/material'
+import { styled } from '@mui/material/styles'
+
+import theme from 'common/theme'
+import PodkrepiIcon from 'components/brand/PodkrepiIcon'
+import CloseModalButton from 'components/common/CloseModalButton'
+import Grid from 'components/recurring-donation/grid/Grid'
+import LinkButton from 'components/common/LinkButton'
+
+export const CloseButton = styled(CloseModalButton)(() => ({
+  marginRight: theme.spacing(1.25),
+}))
+
+export const StyledPodkrepiIcon = styled(PodkrepiIcon)(() => ({
+  margin: theme.spacing(0.7, 0, 1.5, 0),
+}))
+
+export const NavMenuWrapper = styled(Box)(() => ({
+  display: 'flex',
+  justifyContent: 'center',
+  padding: theme.spacing(2),
+  flexDirection: 'column',
+}))
```

Comment:
We can replace this whole styled Box with Stack as it contains all these css attributes as props which makes it really easy to use, also easy to get responsive.